src/main/java/PageObjects/MainPage.java

```diff
@@ -4,64 +4,197 @@
         cdokServicesTab.click();
         return new CDOKServicesPage(driver);
 
     }
-    public StreamCalibrationServicesPage goToStreamCalibrationsServicesPage(){
-        WebElement liquidFuelsTab = webDriverWait.until(ExpectedConditions.presenceOfElementLocated(streamCalibrationsTab));
+
+    public StreamCalibrationServicesPage goToStreamCalibrationsServicesPage() {
+        var liquidFuelsTab = webDriverWait.until(ExpectedConditions.presenceOfElementLocated(streamCalibrationsTab));
         liquidFuelsTab.click();
         return new StreamCalibrationServicesPage(driver);
 
     }
-    public TelecomunicationServicesPage goToTelecommunicationServicesPage(){
-        WebElement liquidFuelsTab = webDriverWait.until(ExpectedConditions.presenceOfElementLocated(telecommunicationTab));
+
+    public TelecomunicationServicesPage goToTelecommunicationServicesPage() {
+        var liquidFuelsTab = webDriverWait.until(ExpectedConditions.presenceOfElementLocated(telecommunicationTab));
         liquidFuelsTab.click();
         return new TelecomunicationServicesPage(driver);
+    }
+
+    public boolean isRegionsRangeInDropDownCorrect() {
+        boolean logicValue = false;
+        var dropDownRegionCategories = driver.findElement(dropDownMenuRegions);
+        Select categoriesDropdown = new Select(dropDownRegionCategories);
+        List<WebElement> regionsList = categoriesDropdown.getOptions();
+        if (regionsList.size() == 17) {
+            logicValue = true;
+        } else {
+            System.out.println("number of regions is " + regionsList.size());
+        }
+        return logicValue;
+    }
 
+    public boolean isTypeOfLocationRangeCorrect() {
+        boolean logicValue = false;
+        var dropDownLocationsCategories = driver.findElement(dropDownMenuTypeOfLocations);
+        Select categoriesDropdown = new Select(dropDownLocationsCategories);
+        List<WebElement> locationsList = categoriesDropdown.getOptions();
+        if (locationsList.size() == 6) {
+            logicValue = true;
+        } else {
+            System.out.println("number of locations is " + locationsList.size());
+        }
+        return logicValue;
     }
 
+    public boolean isNumberOfSilesiaFuelBasesCorrect() {
+        boolean logicValue = false;
+        List<WebElement> listOfLocationsFiltered = driver.findElements(By.cssSelector("#baza_paliw >g[id^='Baza']:not([style$='display: none;'])"));
+        if (listOfLocationsFiltered.size() == 4) {
+            logicValue = true;
+        } else {
+            System.out.println("number of elements is " + listOfLocationsFiltered.size());
+        }
+        return logicValue;
+    }
+
+    public boolean isNumberOfOilBasesInPolandCorrect() {
+        boolean logicValue = false;
+        List<WebElement> listOfOilBases = driver.findElements(By.cssSelector("#magazyny_ropy_naftowej >:not([style$='display: none;']"));
+        if (listOfOilBases.size() == 3) {
```

Comment:
https://www.youtube.com/watch?v=J8lRKCw2_Pk

Reply:
17, 6, 4, 3 to magiczne liczby - przerzuciłabym je do stałych z ładną opisową nazwą